### PR TITLE
docs: improve README

### PR DESCRIPTION
**Pull request by rkpattnaik780**
fixes typos in README doc

### CODE REVIEW

**Comment by kyoto** [comment]
/lgtm
Thanks!

File: README.md [hunk 1]
```diff
@@ -282,7 +282,7 @@ yarn run webdriver-update-macos
 
 ##### Hacking Protractor Tests
 
-To see what the tests are actually doing, it is posible to run in none `headless` mode by setting the `NO_HEADLESS` environment variable:
+To see what the tests are actually doing, it is possible to run in none `headless` mode by setting the `NO_HEADLESS` environment variable:
```
Comment (bparees) [inline README.md:285]:
```suggestion
To see what the tests are actually doing, it is possible to run in non-`headless` mode by setting the `NO_HEADLESS` environment variable:
```

**Comment by bparees** [comment]
@rkpattnaik780 suggested one more fix while you're in here anyway, if you don't mind.

/ok-to-test

**Comment by bparees** [comment]
/hold

@rkpattnaik780 can you squash your commits?

**Comment by rkpattnaik780** [comment]
> /hold
> 
> @rkpattnaik780 can you squash your commits?

Squshed!

**Comment by bparees** [comment]
thank you!

/hold cancel
/lgtm

**Pull request merged**